### DISCUSSION TITLE
Invalid copilot token: missing token: 403 in visual studio code

### DISCUSSION THREAD

**Discussion by jfer86** [community/community#40923] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

when starting visual studio it fails to connect and generates the following error
[INFO] [auth] [2022-12-06T13:01:05.055Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-12-06T13:01:05.056Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

**Comment by luutrutatcavaomaytinh** [comment]
Googlepro

**Comment by iamcjwilliams** [comment]
@jfer86 Sorry for the trouble here, What version of VScode are you using? Could you provide a few more details on how this can be replicated?

**Reply by jfer86** [reply]
Hello, I have already solved it, thanks a lot

**Reply by tamaraantonella** [reply]
How did you do it?

**Reply by tamaraantonella** [reply]
Getting the same 403 error here!

**Reply by haakon-e** [reply]
It should be back online now, check your status here: https://github.com/settings/billing

**Comment by yjm919** [comment]
“If your personal account meets the criteria for a free GitHub Copilot subscription instead of a trial or paid subscription, you will automatically be taken to step 6” , 
 up is step 4,  so  step 5 is to fill your payment infomation,   have you  done????????
because you are the case ****trial or paid subscription ****,.so you should finish  step 5,then use the Copilot.
guys 
haha.
